README.md

```diff
@@ -82,7 +80,7 @@ npm run cypress:run_headless
 
 ## Deployment<a name="deployment"></a>
 
-Project Manager App is deployed to Microsoft Azure: Cloud Computing Services.
+FilmFellow Application is deployed to Microsoft Azure: Cloud Computing Services.
```

Comment:
Varmaan ihan FilmFellow ilman Applicationia riittäis, Project Manager Appissa se App tais olla osa nimeä.